src/optype.rs

```diff
@@ -486,6 +486,7 @@ pub enum OpType {
     /// See [`CustomGate`]
     ///
     ///   [`CustomGate`]: crate::opbox::OpBox::CustomGate
+    #[serde(alias = "Composite")]
```

Comment:
The optype enum said `Composite`, but the class was `CompositeGate`.
https://github.com/CQCL/tket/pull/175/files#diff-76f05b37a15248c7c6dd673f19dc5b8e74829d4d2ae0ad1316c982ab7fd3373fL567-L570
Do you know which one should it be?

I don't have an old json to verify (@johnchildren do you?).
I could also just add both alias.

Reply:
I'm not sure it was ever called `CompositeGate`, but I'd have to look through the tket commit history too check. I think it was called `Composite` with a class of `CompositeGate` but was then renamed to `CustomGate` (both op type and class) later from memory.

Reply:
Sorry yeah see the PR from the issue: https://github.com/CQCL/tket/pull/175/files

It was called `Composite` and was never called `CompositeGate`

Reply:
![image](https://github.com/user-attachments/assets/8232630d-39c5-4e99-9c93-9ea8f679e7b1)

Reply:
Ah, [looks like](https://github.com/CQCL/tket/blob/5055846dd6387c15be3ab3798fd0193f3bdda283/tket/src/Circuit/Boxes.cpp#L582) it should match the `OpType` rather than the class, so `Composite` is correct.